deploy/addon/cellar.md

```diff
@@ -83,12 +83,32 @@ You only need to specify a custom endpoint (eg `cellar-c2.services.clever-cloud.
 ### Node.js
 
 ```javascript
+// Load the AWS SDK for Node.js
 const AWS = require('aws-sdk');
 
-AWS.config.update({accessKeyId: '<cellar_key_id>', secretAccessKey: '<cellar_key_secret>'});
+// Set up config
+AWS.config.update({
+  accessKeyId: '<cellar_key_id>', 
+  secretAccessKey: '<cellar_key_secret>'
+});
 
+// Create S3 service object
 const s3 = new AWS.S3({ endpoint: '<cellar_host>' });
 
+// Create the parameters for calling createBucket
+var bucketParams = {
+  Bucket : '<my-bucket-name>'
+  CreateBucketConfiguration: {
+    LocationConstraint: ''
```

Comment:
No, it's not mandatory.

From what I understand, the aws-sdk js lib adds a zone by default and it doesn't work with ceph/cellar (for bucket creation at least). And AWS users shouldn't have this problem, because they specify the right region every time (it's actually specified in the AWS documentation)

Reply:
What should be the default value for us ?

Reply:
@gaelreyrol I don't remember exactly, but it seems that our default zone is us-something.

I think it's better to ignore the location than to add a us-zone in our documentation, even though it's in France (which may also scare some users)

We may be able to make changes in Ceph later, but for now I think it's fine.